plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -119,7 +119,10 @@ module.exports = class ServiceTree extends Tree {
    */
   findServiceByName(name) {
     return this.findItem(function(item) {
-      return item.getName() === name && item instanceof Service;
+      return (
+        (item instanceof Framework && item.getFrameworkName() === name) ||
```

Comment:
I do agree that they are semantically different, but if anything we should never be using what we previously had since that can also lead to finding the wrong service: 
```js
const serviceTree = new ServiceTree({items:[{
    id: "foo/bar"
},
{
    id: "baz/bar"
}]})

serviceTree.findServiceByName("bar"); // -> finds id = foo/bar but really we wanted id = baz/bar
```